appeals/web/src/server/appeals/appeal-details/inspector-access/inspector-access.service.js

```diff
@@ -0,0 +1,57 @@
+import { convertFromYesNoToBoolean } from '#lib/boolean-formatter.js';
+
+/**
+ *
+ * @param {import('got').Got} apiClient
+ * @param {string} appealId
+ * @param {string} lpaQuestionnaireId
+ * @param {{radio: string, details: string}} updatedLpaInspectorAccess
+ * @returns {Promise<{}>}
+ */
+export function changeLpaInspectorAccess(
+	apiClient,
+	appealId,
+	lpaQuestionnaireId,
+	updatedLpaInspectorAccess
+) {
+	/** @type {{[x: string]: string | boolean}} */
+	const formattedData = {
+		doesSiteRequireInspectorAccess: convertFromYesNoToBoolean(updatedLpaInspectorAccess.radio)
+	};
+	if (convertFromYesNoToBoolean(updatedLpaInspectorAccess.radio)) {
```

Comment:
Very minor, this is being called twice when you could just read the value from `formattedData`